chandra_aca/drift.py

```diff
@@ -173,3 +174,42 @@ def get_aca_offsets(detector, chip_id, chipx, chipy, time, t_ccd):
     ddz = dz_chip - dz_pred
 
     return ddy, ddz
+
+
+def get_web_zero_offset_table():
+    """
+    Get official SOT MP zero offset aimpoint table:
+
+    :returns: zero offset aimpoint table as astropy.Table
+    """
+    ZOA = "https://icxc.harvard.edu/mp/html/aimpoint_table/zero_offset_aimpoints.txt"
+    page = requests.get(ZOA)
+    table_lines = page.content.strip().split("\n")
+    return Table.read(table_lines, format="ascii")
```

Comment:
You can just do `Table.read(page.content, format='ascii')` here.  Stripping and splitting are not required.